components/SponsorsSection.js

```diff
@@ -1,17 +1,17 @@
 import SectionTitle from './SectionTitle'
 
 const sponsorsGold = [
-  'programmer-thai',
-  'sprin3r'
+  { name: 'Thai Programmer Association', image: 'programmer-thai', type: 'gold' },
+  { name: 'SPRINT3r', image: 'sprin3r', type: 'gold' }
 ]
 const sponsorsSliver = [
-  'cleverse',
-  'integenxe',
-  'omise',
-  'pronto',
-  'taskworld',
-  'thinknet',
-  'wongnai'
+  { name: 'Cleverse', image: 'cleverse', type: 'sliver' },
+  { name: 'int’genxe', image: 'integenxe', type: 'sliver' },
+  { name: 'Omise', image: 'omise', type: 'sliver' },
+  { name: 'Pronto Tools', image: 'pronto', type: 'sliver' },
+  { name: 'Taskworld', image: 'taskworld', type: 'sliver' },
+  { name: 'THiNKNET', image: 'thinknet', type: 'sliver' },
+  { name: 'Wongnai', image: 'wongnai', type: 'sliver' }
```

Comment:
I think duplicating `type: 'silver'` in each list item is a bit of an anti-pattern. What you can do instead is to use curried functions.

`renderSponsor` could be written like this:

```js
function createSponsorRenderer (type) {
  return function renderSponsor (detail) {
    return (
      <ItemSponsor
        key={detail.name}
        name={detail.name}
        src={`static/sponsor/${detail.image}.svg`}
        type={type}
      />
    )
  }
}
```

Then it can be used like this:

```js
{sponsorsGold.map(createSponsorRenderer('gold'))}
{sponsorsSliver.map(createSponsorRenderer('silver'))}
```